tsl/test/expected/compression_merge.out

```diff
@@ -271,9 +271,9 @@ SELECT _ts_meta_sequence_num FROM _timescaledb_internal.compress_hyper_10_187_ch
  _ts_meta_sequence_num 
 -----------------------
                     10
-                    20
-                    30
-                    40
+                    10
+                    10
+                    10
```

Comment:
are those all distinct segments? if not something is wrong here. can you improve the test output to include the segment